src/plugins/recurring-issues/index.js

```diff
@@ -21,6 +21,9 @@ The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YY
 
 - [ ] Remove the 'tsc agenda' label on this issue
 - [ ] Review open pull requests and merge any that are [ready](https://eslint.org/docs/maintainer-guide/pullrequests#when-to-merge-a-pull-request)
+- [ ] Release \`@eslint/js\` with to match the upcoming \`eslint\` version in [Jenkins](https://jenkins.eslint.org)
```

Comment:
```suggestion
- [ ] Release \`@eslint/js\` to match the upcoming \`eslint\` version in [Jenkins](https://jenkins.eslint.org)
```

(or maybe something is missing after "with"?)